pyproject.toml

```diff
@@ -63,7 +63,7 @@ reconcile = "databricks.labs.remorph.reconcile.execute:main"
 
 [tool.hatch.envs.default.scripts]
 test        = "pytest --cov src --cov-report=xml tests/unit"
-coverage    = "pytest --cov src tests/unit --cov-report=html"
+coverage    = "pytest --cov src tests --cov-report=html"
```

Comment:
not sure about this one, do we want to run coverage on both unit and integration tests